sync.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# x86_64 or arm64
+[ "$(uname -m)" != "x86_64" ] && USE_ARM64="-arm64"
+
+# security or no security
+[ "$SECURITY_SERVICE_NEEDED" != true ] && USE_NO_SECURITY="-no-secty"
+
+# redis or mongo
+[ "$FOR_REDIS" = true ] && USE_REDIS="-redis"
+
+# nightly or other release
+USE_RELEASE=${RELEASE:-nightly-build}
+if [ "$USE_RELEASE" = "nightly-build" ]; then
+     COMPOSE_FILE="docker-compose-nexus${USE_REDIS}${USE_NO_SECURITY}${USE_ARM64}.yml"
+else
+     COMPOSE_FILE="docker-compose${USE_REDIS}-${USE_RELEASE}${USE_NO_SECURITY}${USE_ARM64}.yml"
+fi
+
+wget -q "https://raw.githubusercontent.com/edgexfoundry/developer-scripts/master/releases/${USE_RELEASE}/compose-files/${COMPOSE_FILE}"
+
+# Use Centos base image instead of Alpine base image for Kong for x86_64 CI
+# due to compatibility issues with Alpine image in CI
+# sed command of MacOS is in different syntax 
+[ "$(uname -m)" = "x86_64" ] && [ "$(uname)" = "Darwin" ] && sed -i '' 's/kong:1.3.0$/kong:1.3.0-centos/' ${COMPOSE_FILE} || sed -i 's/kong:1.3.0$/kong:1.3.0-centos/' ${COMPOSE_FILE}
```

Comment:
I don't think this line does what you want.  if any of the condtions fails (x86_64, Darwin, or sed) it will end up running the Linux sed command instead of the BSD one.   At a minimum, I would make the x86_64 check into an if/then.

Reply:
Changed to:
```
# Use Centos base image instead of Alpine base image for Kong for x86_64 CI
# due to compatibility issues with Alpine image in CI
# sed command of MacOS is in different syntax 
if [ "$(uname -m)" = "x86_64" ] && [ "$(uname)" = "Darwin" ]; then
     sed -i '' 's/kong:1.3.0$/kong:1.3.0-centos/' ${COMPOSE_FILE}
elif [ "$(uname -m)" = "x86_64" ]; then
     sed -i 's/kong:1.3.0$/kong:1.3.0-centos/' ${COMPOSE_FILE}
fi
```